README.md

```diff
@@ -15,8 +15,18 @@ This plugin intends to help you in tracking down problems when you are using css
 where container is the css class that you want to mark as used.
 Add all such classes in the array.
 
+>If you use the `camelCase` option of `css-loader`, you must also enabled it for this plugin
+```js
+/* eslint css-modules/no-unused-class: [2, { camelCase: true }] */
+```
+
 * `css-modules/no-undef-class`: You must not use a non existing class.
 
+>If you use the `camelCase` option of `css-loader`, you must also enabled it for this plugin
+```js
+/* eslint css-modules/no-unused-class: [2, { camelCase: true }] */
```

Comment:
Yeah, we should keep the options near the rules as they are currently,
but also add documentation how to add the rule in the .eslintrc file.